### PR TITLE
feat: message provider

### PR DESCRIPTION
**Pull request by Eghinner**
This one is a test without a provider
WhatsApp_cf94e137AU
This one is a test with a provider, and just in WhatsApp web
chrome_7asmD377Je

### CODE REVIEW

File: src/pages/ShoppingCart/utils/getCustomerServiceLink.ts [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
   phone: string;
   message: string;
 }
 
+// interface Provider {
+//   value: {
+//     commercial_name: string;
+//   };
+// }
+
```
Comment (heliomar-pena) [inline src/pages/ShoppingCart/utils/getCustomerServiceLink.ts]: Please remove comment

**Review by heliomar-pena — APPROVED** [review]
looks good!

File: src/pages/ShoppingCart/utils/getCustomerServiceLink.ts [hunk 2]
```diff
@@ -16,9 +17,14 @@ export const getWebWhatsappLink = ({
   return url;
 };
 
-export const getInvoiceText = (shoppingCartProducts: ShoppingCartProduct[]) => {
+export const getInvoiceText = (
+  shoppingCartProducts: ShoppingCartProduct[],
+  provider: string | undefined
```
Comment (heliomar-pena) [inline src/pages/ShoppingCart/utils/getCustomerServiceLink.ts]: I'm not sure if this works, please test and confirm hehe
```suggestion
  provider?: string
```

Comment (Eghinner) [reply src/pages/ShoppingCart/utils/getCustomerServiceLink.ts]: works, thanks

**Comment by Eghinner** [comment]
@angelik0828 Steps to test

1. Login
2. Go to providers
3. Select 'tribeca'
4. Buy something
5. Go to the cart
6. Click 'Consultar'

**Comment by angelik0828** [comment]
## ✔️ Validation was performed

- When entering supplier
- Product
- Shopping cart
- Consult
- It redirects me to WhatsApp with a message that allows me to make the query known by this means


image
 
image

image

**Review by angelik0828 — APPROVED** [review]

**Pull request merged**